### DISCUSSION TITLE
CORS: ERROR : blocked by CORS policy: Method PATCH is not allowed by Access-Control-Allow-Methods

### DISCUSSION THREAD

**Discussion by Harishguhan21** [directus/directus#21521] [Feature Requests]
### Summary

When i try to call the update API getting the cors error
![Screenshot from 2024-02-22 17-20-40](https://github.com/directus/directus/assets/126090197/141524cf-1f02-4248-b036-abb44e4a8338)

And Also I have configured the cors in .env file as well as 

Here is the screenshot

### Basic Example

   const response = axios.patch(
        `http://192.168.29.233:8055/items/User_details/${editValues[0].id}`,
        loginPayload,
      );

### Motivation

Can anyone tell me why im not able hit the update API without cors error How can i fix the error 

### Detailed Design

In postman its working fine, when i try to hit the API from the browsar facing the isssue

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

![Screenshot from 2024-02-22 17-22-32](https://github.com/directus/directus/assets/126090197/8e12f535-41fd-48c8-a864-e54c1668a2c7)

### Alternatives

![Screenshot from 2024-02-22 17-20-40](https://github.com/directus/directus/assets/126090197/d5e6e356-d359-4ec0-be47-cde030556c75)

### Adoption Strategy

![Screenshot from 2024-02-22 17-20-40](https://github.com/directus/directus/assets/126090197/104c3c0a-6253-432c-92c7-8d1c0a308671)

### Unresolved Questions

![Screenshot from 2024-02-22 17-20-40](https://github.com/directus/directus/assets/126090197/56e9a357-c437-4014-ac6b-eece023d2693)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
